src/main/java/duke/Task.java

```diff
@@ -0,0 +1,109 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import java.util.Locale;
+
+/**
+ * Task class
+ */
+public class Task {
+    protected String task;
+    protected boolean isDone;
+    protected DateTimeFormatter f1 = DateTimeFormatter.ofPattern("d/M/yyyy", Locale.ENGLISH);
+    protected DateTimeFormatter f2 = DateTimeFormatter.ofPattern("yyyy-M-d", Locale.ENGLISH);
+    protected DateTimeFormatter f3 = DateTimeFormatter.ofPattern("d MMM yyyy", Locale.ENGLISH);
+    protected DateTimeFormatter f4 = DateTimeFormatter.ofPattern("d-M-yyyy", Locale.ENGLISH);
```

Comment:
it felt neater to give ` f1` rather than `format1`, which is why i made this choice, do lmk if you still think its more appropriate to do `format1` though

Reply:
As someone who has recently done this increment I know it's because they asked us to accept and display the dates in different formats, but to someone who isn't familiar with the application requirements this might be a little confusing.

What I did was name them something like `UserInputFormat`, `DisplayFormat`, `SaveFormat` etc. This way, it provides a little bit more information on the intended usage of these formats rather than just `format1`, `format2`, etc.

Reply:
okay gets. actually i just put them all in as like kinda edge cases of other date formats being input. like giving it more options to not throw an exception but okay ill see how i can rename them a bit better